src/main/java/com/ombremoon/enderring/common/init/entity/StatusEffectInit.java

```diff
@@ -50,7 +51,7 @@ public class StatusEffectInit {
                     .addAttribute(new AttributeModifier(
                                     UUID.fromString("b9f71f4c-cc64-4b36-b086-7b8bdb40834"),
                                     "CRIMSONSPILL_CRYSTAL",
-                                    30F, AttributeModifier.Operation.ADDITION),
+                                    20F, AttributeModifier.Operation.ADDITION),
```

Comment:
Incorrect number and operation
https://eldenring.wiki.fextralife.com/Crimsonspill+Crystal+Tear